### PR TITLE
bump for library go

### PR DESCRIPTION
**Pull request by deads2k**
This syncs up library-go and api and makes use of them in cluster up

@openshift/sig-master

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -1808,7 +1808,7 @@ imports:
   - pkg/util/proto
   - pkg/util/proto/validation
 - name: k8s.io/kubernetes
-  version: d1702ec28ea20dfa3111587b45875232702414ca
+  version: 48e4040dfae5691fe877037c75ac7c8c061260a7
```
Comment (mfojtik) [inline glide.lock]: @deads2k this bumps because of publisher-bot doing its job?

File: glide.lock [hunk 2]
```diff
@@ -719,7 +719,7 @@ imports:
 - name: github.com/Microsoft/hcsshim
   version: 6ea7fe54f719d95721e7d9b26ac0add224c9b923
 - name: github.com/miekg/dns
-  version: eac804ceef194db2da6ee80c728d7658c8c805ff
+  version: e57bf427e68187a27e22adceac868350d7a7079b
```
Comment (mfojtik) [inline glide.lock]: @deads2k why this bumps?

Comment (deads2k) [reply glide.lock]: > @deads2k why this bumps?

a couple months ago jordan made it float on a version: https://github.com/openshift/origin/blame/master/glide.yaml#L179  Seems ok to me.  We'll run the pin script when we're done

File: glide.lock (hunk 1 above)
Comment (deads2k) [reply glide.lock]: > @deads2k this bumps because of publisher-bot doing its job?

yeah

**Pull request merged**